frontend/views/jobPreviouslyExecuted.py

```diff
@@ -12,7 +12,9 @@
 from utils.docker_utils import runConfigAssessmentTool, isDocker
 
 
-def jobPreviouslyExecuted(client: APIClient, jobName: str, debug: bool, concurrentConnections: int, platformStr: str, tag: str):
+def jobPreviouslyExecuted(
+    client: APIClient, jobName: str, debug: bool, concurrentConnections: int, password_dynamically: str, platformStr: str, tag: str
```

Comment:
Same as above: I think propagating this variable down is needless? It is set in the jobPreviously/NotYetExecuted elements.